DataOnion/Extensions.cs

```diff
@@ -1,47 +1,95 @@
 using System.Data.Common;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
-using DataOnion.Config;
 using DataOnion.db;
 
 namespace DataOnion
 {
-    public static class DataHelperExtensions
+    public interface IFluentDatabaseOnion
```

Comment:
I'm not familiar enough with C# to say whether this would be possible, but in Swift I would probably do:
```swift
public typealias IFluentDatabaseOnion = IFluentDapperSetup & IFluentEfCoreSetup
```
And I believe TypeScript has something similar. Even if that doesn't exist, is there a reason the declarations are duplicated rather than saying
```cs
public interface IFluentDatabaseOnion : IFluentDapperSetup, IFluentEfCoreSetup {}
```
?

Reply:
you can just compose it from both interfaces as you say

why i didn't, not sure probably just didnt think about it you can do it if you want